content/information-distribution/node.md

```diff
@@ -44,81 +44,132 @@ This ping **must** take the form of a Matrix timeline event of the
+following structure:
+
+| Parameter    | Type   | Req. | Description                                                   |
+|:-------------|:-------|:----:|:--------------------------------------------------------------|
+| `entry_node` | `bool` |  x   | Whether the node is an entry node for this federation (room). |
 
-The example above shows the content of the `m.room.aliases` state event for the
-node which uses `example.com` as its server name. Please note that an instance
-of a `m.room.aliases` state event only lists the aliases created by a single
-node (hence both aliases using the same server name in the example), which means
-that the complete list of aliases for the Matrix room can only be retrieved by
-processing every `m.room.aliases` state event that hasn't been made obsolete by
-a more recent `m.room.aliases` state event for the same node.
+### Example
 
-Client implementations **must** process these lists of aliases to generate a map
-associating all of the entry nodes for this Matrix room with the aliases they
-provide for this room.
+Let's consider a node that has `example.com` as its server name, and wants to
+act as an entry node for the federation which Matrix room ID is
+`!Nei7aeg5aefub:informo.network`.
 
-### Reaching an entry node
+This node **must** serve a file at the address
+`https://example.com/.well-known/informo/info` with the following content:
+
+```
+{
+    "!Nei7aeg5aefub:informo.network": {
+        "entry_node": true
+    }
+}
+```
 
-Because processing the list of aliases gives only the nodes' server names, and
-not their FQDN, a node might not be reachable at the address defined by its
+### Client implementations behaviour
+
+#### Computing a list of entry nodes
+
+Client implementations **must** provide user with a list of known entry nodes,
+for at least one federation, that's up to date at the time of the
+implementation's release. Client implementations **must** also allow users to
+use an unknown node as an entry node (provided that this node implements the
+requirements to be an entry node).
+
+Once a federation is joined (see the section [below]({{<relref
+"#joining-a-federation">}})), client implementations **must** retrieve and save
+the list of joined nodes in the federation. This can be done by retrieving the
+list of [joined
+users](https://matrix.org/docs/spec/client_server/r0.4.0.html#get-matrix-client-r0-rooms-roomid-joined-members)
+for the federation, then processing this list in order to only keep the server
+part of the Matrix user IDs (i.e. what's after the colon (`:`)). Client
+implementations **must** save this list in case the node they're currently using
+becomes unreachable.
+
+Client implementations **should** weight this list according to the number of
+Matrix users belonging to a given node in a federation. The thinking behind this
+is that a node with many users in a federation is more likely to be an entry
+node for this federation than a node with only a few users in it.
```

Comment:
Good point.

Reply:
I don't see any good solution either. I'd rather remove this part and let the implementation decide, since it's not really that important anyway (IMO the important thing is ordering by trust counts)

Reply:
Yeah, that makes sense.

Reply:
Fixed in 2efd4af